.github/workflows/main.yml

```diff
@@ -103,8 +103,33 @@ jobs:
       ##
       ## Run TLC tests.
       ##
+      ## The following process first compiles and packages (dist) the tla2tools.jar file. Next, it 
+      ## runs the test suite in parallel, aborting the run and failing the workflow upon the first
+      ## test failure (test.halt=true). Regardless of any test failures, a JUnitReport is generated.
+      ## Both the report and the test results (in XML format) are uploaded as workflow artifacts for
+      ## potential local download and further analysis. Additionally, mikepenz/action-junit-report
+      ## publishes the stack traces from the test results. If desired, the test.halt setting can be
+      ## removed to allow the test suite to run to completion.
+      ## (The same sequence of steps is executed by pr.yml)
+      ## 
     - name: Run TLC tests
       run: ant -f tlatools/org.lamport.tlatools/customBuild.xml -Dtest.halt=true compile compile-test test
```

Comment:
Should this line include the `info` target to match the one in `pr.yml`?

Reply:
Good point, wasn't there before but has been added in https://github.com/tlaplus/tlaplus/pull/954/commits/608a4b8e5657668d63c9cbe77ad53c71693e4228